sim/index.html

```diff
@@ -55,67 +55,67 @@
 
 				<div id="section_r">
 
-					R<sub>0</sub> is <span id="label_p_r0" toFixed="2"></span>
+					R<sub>0</sub> è <span id="label_p_r0" toFixed="2"></span>
 					<canvas class="r_canvas" id="canvas_r0"></canvas>
 					<!--input class="sim_input" type="range" id="p_r0" min="0" max="6" step="0.01" disabled-->
 					<br>
 
 					<span id="label_s">
-						% of people who are <i>NOT</i> <icon s></icon>
+						% di persone che <i>NON</i> sono <icon s></icon>
 						<input class="sim_input" type="range" id="p_s" min="0" max="1" step="0.001" value="0" disabled>
 						<div class="herd"></div>
 						<br>
 					</span>
 
 					<span id="int_block_0">
-						Increased Hygiene
+						Maggiore igiene
 						<br>
 						<input class="sim_input recordable" type="range" id="p_hygiene" min="0" max="1" step="0.001" value="0">
 						<br>
 					</span>
 					<span id="int_block_1">
-						Physical Distancing
+						Distanziamento Sociale
```

Comment:
Distanziamento fisico?

Reply:
I used "distanziamento sociale" because it's the common term heard on the tv/media. Anyway I can change, let me know your thoughts :)

Reply:
I agree with @simonefalcini, it is more widely used.

Reply:
I will merge the PR for now, if you @pietroppeter have strong opinions about that we can discuss that further in #3.